sdk/identity/identity/rollup.base.config.js

```diff
@@ -93,7 +94,7 @@ export function browserConfig(test = false, production = false) {
   };
 
   if (test) {
-    baseConfig.input = "dist-esm/test/**/*.spec.js";
+    baseConfig.input = "dist-esm/test/*.spec.js";
```

Comment:
I kinda like **/* here - don't have to make changes later if we move tests into subdirs, and there doesn't seem like a risk of slurping up too many files.

Reply:
The reason why I removed the `*/` is so that I could have a separate `node/` subfolder for specs that only run on Node.js.  I've seen this pattern used in the storage libraries.  If you know of a better pattern for avoiding spec files that only work on Node.js I'd certainly be happy to use it!

Reply:
Ahh yeah that makes sense. Carry on!